### PR TITLE
Remove selected query afte the click the score button

### PR DESCRIPTION
**Pull request by EverLoSa**
Description:
Scoring an URL in the Scoring Panel does not removes the URL when clicking the Score button

### CODE REVIEW

File: spot-oa/ui/dns/js/components/ScoreNotebook.react.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ var ScoreNotebook = React.createClass({
       dataScored.push([dstIp, query, rating]);
     }
 
-    this.removeSelected([dstIp, quickIpScoring]);
+	this.removeSelected([dstIp, quickIpScoring,query]);
```
Comment (rabarona) [inline spot-oa/ui/dns/js/components/ScoreNotebook.react.js]: Is the extra indentation intended?

**Comment by raypanduro** [comment]
+1

**Comment by natedogs911** [comment]
+1

**Comment by vgonzale78** [comment]
+1

**Pull request closed without merging**